paper/paper.md

```diff
@@ -67,13 +84,22 @@ for species, agents in grouped:
 The framework automatically handles agent lifecycle management, including:
 - Unique ID assignment
 - Agent registration and removal
-- Type-based organization
+- Agent subclass-based organization
 - Efficient collective operations
 
-### Spatial modeling
-Mesa supports multiple approaches to spatial modeling:
 
-1. Grid-based spaces:
+
+### Spaces
+Mesa offers a variaty of spaces within which agents can be located. A basic distinction is between discrete or cell-based spaces, and continous space. In discrete spaces, the space consists of a collection of cells and agents occupy a cell. Examples of this include orthogonal grids, hexgrids, voroinoi meshes, or networks. In a continous space, in constrast, an agent has a location.
+
+Mesa comes with a wide variety of discrete spaces, including OrthogonalMooreGrid, OrthogonalVonNeumanGrid, Hexgrid, Network, and VoroinoiMesh. These are all implemented using a doubly linked data structure where each cell has connections to its neighboring cells. The different discrete spaces differ with respect to how they are "wired-up", but the API is uniform across all of them.
+
+Mesa also offers 3 subclasses of the Agent class that are designed to be used in conjunction with these discrete spaces: FixedAgent, CellAgent, and Grid2DMovingAgent. FixedAgent is assigned to a given cell and can access this cell via `self.cell`. However, once assigned to a given cell, it can not be moved. A CellAgent, like a FixedAgent, has access to the cell it currently occupies. However, it can update this attribute making it possible to move around. A Grid2DMovingAgent extends CellAgent by offering a move method with short hand for the direction of movement.
+
```

Comment:
Personally, since all this is experimental, I would save it for a Mesa 4 paper when it's stable.

Reply:
Or make it significantly shorter, as a "we're working on"

Reply:
I disagree. From 3.2 onwards, cell spaces are stable, and old-style spaces raise a pending deprecation warning. It is plausible that by the time this paper is accepted 3.2 is out.